dcos/dcos-ui
# DCOS-50468: Nodes Table Loader does not behave as expected

Change the initial value for nodes to null
to show loader while data is loading.

Closes https://jira.mesosphere.com/browse/DCOS-50468

## Testing
1. Go to Nodes tab
2. Refresh the page
3. Verify that an empty table is not visible as data is loading

## Trade-offs
None.

## Dependencies
None.

## Screenshots

### Before
![Peek 2019-03-20 11-42](https://user-images.githubusercontent.com/40791275/54688799-03bc7180-4b27-11e9-94d2-1eae943fd5c9.gif)

### After
![Peek 2019-03-20 15-41](https://user-images.githubusercontent.com/40791275/54688807-07e88f00-4b27-11e9-8e31-9218e400b752.gif)
